wal/src/manager.rs

```diff
@@ -119,12 +119,28 @@ pub mod error {
     define_result!(Error);
 }
 
-use common_types::{MAX_SEQUENCE_NUMBER, MIN_SEQUENCE_NUMBER};
+use common_types::{table::TableId, MAX_SEQUENCE_NUMBER, MIN_SEQUENCE_NUMBER};
 pub use error::*;
 
 pub type RegionId = u64;
 pub const MAX_REGION_ID: RegionId = u64::MAX;
 
+/// Used to wals of the table.
+#[derive(Debug, Clone, Copy, Hash, Eq, PartialEq, Ord, PartialOrd)]
+pub struct WalLocation {
```

Comment:
`Location` seems enough. After all, there is no such a struct called `WalRegionId`.

Reply:
It seems reasonable in wal crate. But I think it should be renamed to `WalLocation` while use in other modules by `use as`.

Reply:
`Location` should not be a usual name. In my memory, there is no struct called `Location` now.

Reply:
> `Location` should not be a usual name. In my memory, there is no struct called `Location` now.

I mean `Location`s in other crate, such as `std::panic::Location`, `snafu::Location`. And If just say `Location`, I will think it is used to locate the table in database first but not the location in wal module.

Reply:
Done.